.github/workflows/retry_test.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       ${{ 
         github.event.workflow_run.conclusion == 'failure' &&
         github.repository == 'vercel/next.js' &&
+        github.event.workflow.name != 'build-and-deploy' &&
```

Comment:
I can this is not the correct value to check, retries for `build-and-test` will still work.